packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -342,6 +343,23 @@ class MenuContent extends React.Component {
     );
   }
 
+  ariaDescribedByHandle(value, index) {
```

Comment:
This function can be simplified to 
```js
    ariaDescribedByHandle(value, index) {
      if (!MenuUtils.isMac()) {
        let ariaDescribedByValue = (value.props.ariaDescribedBy) ? `, ${value.props.ariaDescribedBy}` : '';
        if (!this.props.index && this.props.showHeader && index === 0) {
          return `${this.menuTopHeaderId}${ariaDescribedByValue}`;
        }
      }
      return value.props.ariaDescribedBy;
    }
```

Reply:
Updated - https://github.com/cerner/terra-framework/pull/1841/commits/37d673e339b59651df28ce8ec477056360fdfbbc